server/docs/Dockerfile.build

```diff
@@ -0,0 +1,18 @@
+FROM alpine:3.6
+
+RUN apk --update add \
+  ruby ruby-io-console \
+  ruby-dev \
+  build-base \
+  libffi-dev libxml2-dev libxslt-dev \
+  nodejs
+RUN gem install bundler --no-ri --no-rdoc
+
+ADD Gemfile Gemfile.lock /home/build/
+WORKDIR /home/build
+
+RUN bundle config build.nokogiri --use-system-libraries
+RUN bundle install
+
+VOLUME /home/build # expect -v $PWD:/home/build
+CMD middleman build
```

Comment:
Doesn't work for whatever reason, fails with

`$ docker build -t kontena/server-docs.build -f Dockerfile.build . && docker run --rm -it -v $PWD:/home/build kontena/server-docs.build`

```
/home/build not found
```